market-test/src/test/java/com/axonivy/market/test/CreateBundleTest.java

```diff
@@ -0,0 +1,33 @@
+package com.axonivy.market.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.axonivy.market.CreateBundle;
+
+public class CreateBundleTest
```

Comment:
i don't know if the build is executing and collecting tests?

Reply:
yet it is. :+1: